pkg/odo/cli/component/create.go

```diff
@@ -259,18 +262,18 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 		co.interactive = true
 	}
 
-	co.localConfigInfo, err = config.NewLocalConfigInfo(co.componentContext)
+	co.localConfig, err = config.NewLocalConfigInfo(co.componentContext)
```

Comment:
there is a `LocalConfig` struct as well in `config` package and hence this could be confusing, we can consider `lci` if `localConfigInfo` is too long.

Reply:
Updating